alpha/lib/model/DeliveryProfileLiveAppleHttp.php

```diff
@@ -324,5 +256,26 @@ private function buildFlavorBitrateInfoArray($primaryServerStreams)
 		}
 		return $flavorBitrateInfo;
 	}
+	
+	private function isAudioStream($primaryServerStreams, $backupServerStreams)
+	{
+		if(count($primaryServerStreams))
+		{
+			/* @var $kLiveStreamParams kLiveStreamParams */
+			$kLiveStreamParams = reset($primaryServerStreams);
+			if(!$kLiveStreamParams->getHeight() && !$kLiveStreamParams->getWidth())
```

Comment:
in this case 0 is not a bool - should not be handled as such 
would use getHeight == 0 && getWidth ==  0